mrchecker-framework-modules/mrchecker-webapi-module/src/main/java/com/capgemini/mrchecker/webapi/pages/httbin/SimpleXMLPage.java

```diff
@@ -0,0 +1,45 @@
+package com.capgemini.mrchecker.webapi.pages.httbin;
+
+import com.capgemini.mrchecker.webapi.core.BasePageWebAPI;
+import com.capgemini.mrchecker.webapi.core.base.driver.DriverManager;
+import com.capgemini.mrchecker.webapi.pages.environment.GetEnvironmentParam;
+
+import io.restassured.response.Response;
+import org.json.XML;
```

Comment:
Import is not used here
You could move logic that deals with XML document parsing from Test class to Page class or some helper class (reading XML document logic in setup method in test class)

Reply:
fixed. XML logic moved to SimpleXMLParser